xpcPlugin/MessageHandlers.cpp

```diff
@@ -626,6 +630,89 @@ namespace XPC
 			}
 		}
 	}
+	
+	void MessageHandlers::HandleGetT(const Message& msg)
+	{
+		const unsigned char* buffer = msg.GetBuffer();
+		std::size_t size = msg.GetSize();
+		if (size != 30)
+		{
+			Log::FormatLine(LOG_ERROR, "GTER", "Unexpected message length: %u", size);
```

Comment:
"GTER" should be "MSGH". The main point of those tags is to quickly indicate which source file generated a particular line in the log.

Reply:
I should note that I myself have hardly been consistent on this point, so it's hardly your fault. That said, I'd like to move towards one tag per file for logging.

Reply:
All functions in MessageHandler have their name in that tag. If using MSGH everywhere, one would loose any pointer which functionality is affected? Can change it, but to me the current way makes more sense.

Reply:
Alright, forget changing it to `MSGH`. Should it at least match the message name (i.e. `GETT` instead of `GTER`)?

Reply:
I'll change it to GETT.